java/src/main/java/com/google/appengine/tools/mapreduce/GcpCredentialOptions.java

```diff
@@ -33,14 +44,44 @@
   }
 
   //helper util; consider moving to GCPUtils class, or something ...
-  static Storage getStorageClient(GcpCredentialOptions gcpCredentialOptions) {
-    Credentials credentials =
-      gcpCredentialOptions.getServiceAccountCredentials().map(c -> (Credentials) c)
-        .orElseGet(() -> StorageOptions.getDefaultInstance().getCredentials());
+  static Storage getStorageClient(@Nullable GcpCredentialOptions gcpCredentialOptions) {
+    Credentials credentials = determineCredentials(gcpCredentialOptions)
+      .orElseGet(() -> StorageOptions.getDefaultInstance().getCredentials());
 
     return StorageOptions.newBuilder()
       .setCredentials(credentials)
       .build().getService();
+  }
+
+
+
+  static Bigquery getBigqueryClient(@Nullable GcpCredentialOptions gcpCredentialOptions) {
+
+    //weird, but this is what FW was using previously ...
+    HttpTransport transport = new NetHttpTransport(); //new UrlFetchTransport();
```

Comment:
Ok, so I think that this needs to be change as same time as https://github.com/Worklytics/evalengin/pull/4105 because with the new libraries there is no need to use `HttpTransport` and `JsonFactory` neither

Reply:
yeah - we need analogous change here ... or not, because actually we don't even use the BQ jobs included in the FW, I think.